src/hotspot/share/services/rsswatch.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 1999, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright should not include 1999.

Reply:
Sorry, forgot to commit that change appearantly.